spec/models/ability_spec.rb

```diff
@@ -171,6 +171,35 @@
         end
       end
 
+      describe "just orga members, team\'s supervisor and team\'s students should be able to see offered conference for a team" do
+        let(:user) { FactoryGirl.create(:student)}
+
+        context 'if it is an student of another team' do
```

Comment:
For the sake of readability, I would try to come up with something starting with either _"when"_ or "with"_ for the `context` hooks - it really helps understanding what's going on. Also: using "it" is a bit ambiguous here...

How about something like:
```ruby
context "when the user is a student on another team" do
  # do stuff
```

_(this also goes for the other contexts you define here)_